addons/web/static/src/views/fields/properties/property_definition.js

```diff
@@ -280,6 +280,20 @@ export class PropertyDefinition extends Component {
         this.state.propertyDefinition = propertyDefinition;
     }
 
+    /**
+     * We activate / deactivate the property in the kanban view.
+     *
+     * @param {boolean} newValue
+     */
+    onViewInKanbanChange(newValue) {
+        const propertyDefinition = {
+            ...this.state.propertyDefinition,
+            view_in_kanban: newValue,
```

Comment:
Not sure for the "view in kanban", but ok, I suppose a bit of control is welcomed

Reply:
the functional argument is that we might have a lot of properties, and showing all of then can be a bit messy

Reply:
OK for me :)